README.md

```
@@ -688,13 +693,138 @@ collection2.ReplaceOne((Predicate<dynamic>)(e => e.id == 11), dynamicUser);
+{
+    { "Age", 41 },
+    { "Name", "James" }
+};
+var jsonString = JsonSerializer.Serialize(patchData);
+var options = new JsonSerializerOptions { Converters = { new SystemExpandoObjectConverter() } };
+dynamic patchExpando = JsonSerializer.Deserialize<ExpandoObject>(jsonString, options);
+
+await collection.UpdateOneAsync(e => e.Id == 12, patchExpando);
+```
+
+#### 4. Date Handling with Dynamic Types
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)